cime_config/user_nl_blom_outvol_reduced

```diff
@@ -0,0 +1,534 @@
+  LYR_NATCALC       = 0, 0
+  LYR_NATPH         = 0, 0
+  LYR_NATOMEGAC     = 0, 0
+  LYR_NATOMEGAA     = 0, 0
+  LYR_DIC13         = 0, 0
+  LYR_DIC14         = 0, 0
+  LYR_D13C          = 0, 0
+  LYR_D14C          = 0, 0
+  LYR_BIGD14C       = 0, 0
+  LYR_POC13         = 0, 0
+  LYR_DOC13         = 0, 0
+  LYR_CALC13        = 0, 0
+  LYR_PHYTO13       = 0, 0
+  LYR_GRAZER13      = 0, 0
+  LYR_CFC11         = 0, 0
+  LYR_CFC12         = 0, 0
+  LYR_SF6           = 0, 0
+  LYR_NOS           = 0, 0
+  LYR_WPHY          = 0, 0
+  LYR_WNOS          = 0, 0
+  LYR_EPS           = 0, 0
+  LYR_ASIZE         = 0, 0
+  LYR_BROMO         = 0, 0
+  LYR_SHELFAGE      = 0, 0
+  LYR_DP@diabgc     = 0, 0
+
+! Save level-fields only annually
+
+  LVL_AGG_AVDC      = 0, 0
+  LVL_AGG_AVDP      = 0, 0
+  LVL_AGG_AVRHOP    = 0, 0
+  LVL_AGG_B         = 0, 0
+  LVL_AGG_DF        = 0, 0
+  LVL_AGG_DMAX      = 0, 0
+  LVL_AGG_STICK     = 0, 0
+  LVL_AGG_STICKF    = 0, 0
+  LVL_AGG_VPOR      = 0, 0
+  LVL_AGG_VRHOF     = 0, 0
+  LVL_AGG_WS        = 0, 0
+  LVL_PHYTO         = 0, 2
+  LVL_GRAZER        = 0, 2
+  LVL_DOC           = 0, 2
+  LVL_PHOSY         = 0, 2
+  LVL_PHOSY_NH4     = 0, 0
+  LVL_PHOSY_NO3     = 0, 0
+  LVL_PHOSPH        = 0, 2
+  LVL_OXYGEN        = 0, 4
+  LVL_IRON          = 0, 2
+  LVL_ALKALI        = 0, 2
+  LVL_ANH4          = 0, 2
+  LVL_ANMX_N2_PROD  = 0, 0
+  LVL_ANMX_OM_PROD  = 0, 0
+  LVL_ANO2          = 0, 2
+  LVL_ANO3          = 0, 2
+  LVL_SILICA        = 0, 2
+  LVL_DIC           = 0, 2
+  LVL_POC           = 0, 2
+  LVL_CALC          = 0, 2
+  LVL_OPAL          = 0, 2
+  LVL_CO3           = 0, 2
+  LVL_N2O           = 0, 2
+  LVL_PH            = 0, 2
+  LVL_OMEGAC        = 0, 2
+  LVL_OMEGAA        = 0, 2
+  LVL_PREFO2        = 0, 4
+  LVL_O2SAT         = 0, 4
+  LVL_PREFPO4       = 0, 2
+  LVL_PREFALK       = 0, 2
+  LVL_PREFDIC       = 0, 2
+  LVL_PREFSILICA    = 0, 2
+  LVL_REMIN_AEROB   = 0, 2
+  LVL_REMIN_SULF    = 0, 2
+  LVL_DICSAT        = 0, 2
+  LVL_NATDIC        = 0, 0
+  LVL_NATALKALI     = 0, 0
+  LVL_NATCO3        = 0, 0
+  LVL_NATCALC       = 0, 0
+  LVL_NATPH         = 0, 0
+  LVL_NATOMEGAC     = 0, 0
+  LVL_NATOMEGAA     = 0, 0
+  LVL_DIC13         = 0, 0
+  LVL_DIC14         = 0, 0
+  LVL_D13C          = 0, 0
+  LVL_DENIT_N2O     = 0, 0
+  LVL_DENIT_NO2     = 0, 0
+  LVL_DENIT_NO3     = 0, 0
+  LVL_DNRA_NO2      = 0, 0
+  LVL_DYNVIS        = 0, 0
+  LVL_POC13         = 0, 0
+  LVL_DOC13         = 0, 0
+  LVL_CALC13        = 0, 0
+  LVL_PHYTO13       = 0, 0
+  LVL_GRAZER13      = 0, 0
+  LVL_CFC11         = 0, 0
+  LVL_CFC12         = 0, 0
+  LVL_SF6           = 0, 0
+  LVL_NOS           = 0, 0
+  LVL_NITR_N2O_PROD = 0, 0
+  LVL_NITR_NH4      = 0, 0
+  LVL_NITR_NH4_OM   = 0, 0
+  LVL_NITR_NO2      = 0, 0
+  LVL_NITR_NO2_OM   = 0, 0
+  LVL_WPHY          = 0, 0
+  LVL_WNOS          = 0, 0
+  LVL_EPS           = 0, 0
+  LVL_ASIZE         = 0, 0
+  LVL_BROMO         = 0, 0
+  LVL_SHELFAGE      = 0, 0
```

Comment:
I am wondering, if we could/should add the shelfage tracer for spinup - to get information on it during pi and potentially, how that changes over hist+scenario.

Reply:
But this is only the output setting. We will certainly  switch on the output for the piControl and all subsequent simulations, but here the question is whether the output would be useful for spinup? IMO this would only be the case if this could provide information for tuning purposes.

Reply:
I agree that we can keep shelfage off for the spinup. - would be nice to switch it on, when starting pi-simulations (that need to be long enough to reach steady state in age - which should be about 100yrs+).